src/routes/docs/tutorials/+page.svelte

```diff
@@ -54,7 +54,7 @@
                     <li class="is-mobile-col-span-2">
                         <a href="/docs/tutorials/vue" class="aw-card is-normal">
                             <header class="u-flex u-cross-center u-gap-4">
-                                <img src="/images/platforms/{$themeInUse}/vue.svg" alt="" />
+                                <img src="/images/platforms/{$themeInUse}/vue.svg" />
```

Comment:
Why was alt removed? An empty alt signifies the image is only decorative, which is desired in this case.